scripts/pmm2_service_check.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
This probably works, but Ansible also has "assert" module so you can put simple checks inside them directly in the playbook or tasks.
https://docs.ansible.com/ansible/latest/modules/assert_module.html
Now I don't have our example because we didn't use this at all, but probably we should have.

Reply:
I used it, could you please review again